DotNetRu.Clients.Portable/Helpers/ParseXmlHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.IO;
+using System.Xml;
+using System.Xml.Serialization;
+
+namespace XamarinEvolve.Clients.Portable.Helpers
+{
+    internal static class ParseXmlHelper
+    {
+
+        public static Stream ToStream(this string @this)
+        {
+            var stream = new MemoryStream();
+            var writer = new StreamWriter(stream);
+            writer.Write(@this);
+            writer.Flush();
+            stream.Position = 0;
+            return stream;
+        }
+
+
+        public static T ParseXml<T>(this string @this) where T : class
+        {
+            var reader = XmlReader.Create(@this.Trim().ToStream(), new XmlReaderSettings() { ConformanceLevel = ConformanceLevel.Document });
```

Comment:
You can just use XmlReader.Create(new StringReader(YourStringValue));
So ToStream() method is not needed

Reply:
I just copy this class from CPOD 🤔

Reply:
@bruce-willis then I've overlooked it there. But let's improve the code here :)